pkg/cmd/taskrun/taskrun.go

```diff
@@ -42,5 +42,6 @@ func Command(p cli.Params) *cobra.Command {
 
 	flags.AddTektonOptions(cmd)
 	cmd.AddCommand(listCommand(p))
+	cmd.AddCommand(logCommand(p))
```

Comment:
please consider .. 
```
cmd.AddCommand(
  listCommand(p),
  logCommand(p),
)
```

Reply:
This is really cool!

Reply:
I think you can do "method chaining" too with cobra?